### ISSUE TITLE
. removed from @INC

### ISSUE THREAD

**Issue by eserte** [mackee/App-Prove-Plugin-MySQLPool#5]
In perl 5.26 the dot is not anymore in `@INC`. This is causing the test suite to fail:
```
Can't locate t/Util.pm in @INC (you may need to install the t::Util module) (@INC contains: /home/cpansand/.cpan/build/2017051306/App-Prove-Plugin-MySQLPool-0.06-0/blib/arch /home/cpansand/.cpan/build/2017051306/App-Prove-Plugin-MySQLPool-0.06-0/blib/lib /opt/perl-5.26.0-RC1/lib/site_perl/5.26.0/x86_64-linux /opt/perl-5.26.0-RC1/lib/site_perl/5.26.0 /opt/perl-5.26.0-RC1/lib/5.26.0/x86_64-linux /opt/perl-5.26.0-RC1/lib/5.26.0) at t/01_env_dsn.t line 7.
BEGIN failed--compilation aborted at t/01_env_dsn.t line 7.
t/01_env_dsn.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
Can't locate t/Util.pm in @INC (you may need to install the t::Util module) (@INC contains: /home/cpansand/.cpan/build/2017051306/App-Prove-Plugin-MySQLPool-0.06-0/blib/arch /home/cpansand/.cpan/build/2017051306/App-Prove-Plugin-MySQLPool-0.06-0/blib/lib /opt/perl-5.26.0-RC1/lib/site_perl/5.26.0/x86_64-linux /opt/perl-5.26.0-RC1/lib/site_perl/5.26.0 /opt/perl-5.26.0-RC1/lib/5.26.0/x86_64-linux /opt/perl-5.26.0-RC1/lib/5.26.0) at t/02_prepare.t line 8.
BEGIN failed--compilation aborted at t/02_prepare.t line 8.
t/02_prepare.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

A possible fix is to add something like `use lib ".";` on top of the failing scripts.

**Comment by mackee** [comment]
@eserte Thanks your report. fixed it by #6. I release 0.07 to CPAN. This version contain fixed this problem.

**Comment by eserte** [comment]
Confirmed, the fix works with perl 5.26.0-RC1.
